parquet_file/src/chunk.rs

```diff
@@ -112,22 +110,20 @@ pub struct ParquetChunk {
 impl ParquetChunk {
     /// Creates new chunk from given parquet metadata.
     pub fn new(
-        file_location: Path,
+        path: ParquetFilePath,
         iox_object_store: Arc<IoxObjectStore>,
         file_size_bytes: usize,
         parquet_metadata: Arc<IoxParquetMetaData>,
         table_name: Arc<str>,
         partition_key: Arc<str>,
         metrics: ChunkMetrics,
     ) -> Result<Self> {
-        let schema = parquet_metadata.read_schema().context(SchemaReadFailed {
-            path: &file_location,
-        })?;
+        let schema = parquet_metadata
+            .read_schema()
+            .context(SchemaReadFailed { path: path.clone() })?;
```

Comment:
Ideally `path` implements `Into` and this works:

```suggestion
            .context(SchemaReadFailed { path })?;
```

Otherwise:

```suggestion
            .with_context(|| SchemaReadFailed { path: path.clone() })?;
```